apps/web/pages/auth/setup/steps/AdminUser.tsx

```diff
@@ -2,6 +2,7 @@ import { zodResolver } from "@hookform/resolvers/zod";
 import classNames from "classnames";
```

Comment:
These files should not live here:

![image](https://user-images.githubusercontent.com/3504472/214646092-758d3106-481b-4582-9a85-75bbf3bb3131.png)

These should not be valid URLs

http://localhost:3000/auth/setup/steps/AdminUser